nomad/job_endpoint.go

```diff
@@ -737,25 +745,32 @@ func (j *Job) BatchDeregister(args *structs.JobBatchDeregisterRequest, reply *st
 
 		// Create a new evaluation
 		eval := &structs.Evaluation{
-			ID:          uuid.Generate(),
-			Namespace:   jobNS.Namespace,
-			Priority:    priority,
-			Type:        jtype,
-			TriggeredBy: structs.EvalTriggerJobDeregister,
-			JobID:       jobNS.ID,
-			Status:      structs.EvalStatusPending,
+			ID:             uuid.Generate(),
+			Namespace:      jobNS.Namespace,
+			Priority:       priority,
+			Type:           jtype,
```

Comment:
Noticed that previous code sets priority and type from the job - but `Deregister` method doesn't.  Should we do that there as well?